styles/doc.css

```diff
@@ -1,92 +1,116 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap");
 
-
-:root{
-    --background-color:#F9F9F9;
+:root {
+  --primary-color: #456568;
+  --secondary-color: #f9f9f9;
+  --attention: red;
 }
 
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
-    font-family: 'Roboto', sans-serif;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
 }
 
-li{
-    list-style: none;
+li {
+  list-style: none;
 }
 
-a{
-    text-decoration: none;
+a {
+  text-decoration: none;
 }
-.nav-container{
-    display: flex;
-    align-items: center;
-    background-color: #456268;
-    font-family: 'Roboto', sans-serif;
-    font-weight: bold;
+.nav-container {
+  display: flex;
+  align-items: center;
+  background-color: var(--primary-color);
+  font-family: "Roboto", sans-serif;
+  font-weight: bold;
 }
-.branding{
-    display: flex;
-    align-items: center;
-    /* gap:1rem; */
-    margin-left: 2rem;
-    padding: 1rem;
-    
+.branding {
+  display: flex;
+  align-items: center;
+  /* gap:1rem; */
+  margin-left: 2rem;
+  padding: 1rem;
 }
-.brand_image{
-    width: 3.5rem;
-    height: 3.5rem;
+.brand_image {
```

Comment:
display: flex, align-items: center is repeated in both branding and nav-container classes maybe u could make a utilitity class for them and re-use it if u r going to use them many times